demo/sgx-sdk-demo/Dockerfile

```diff
@@ -44,13 +44,14 @@ RUN wget ${SGX_SDK_URL} \
  && echo "yes" | ./$SGX_SDK_INSTALLER \
  && rm $SGX_SDK_INSTALLER
 
-RUN git clone -b $DCAP_VERSION https://github.com/intel/SGXDataCenterAttestationPrimitives.git
-
 RUN cd sgxsdk/SampleCode/SampleEnclave \
     && . /opt/intel/sgxsdk/environment \
     && make \
     && cd -
 
+RUN wget https://github.com/intel/SGXDataCenterAttestationPrimitives/archive/$DCAP_VERSION.tar.gz && \
```

Comment:
I was going to hit approve already but recalled one more thing: we have the similar pattern used in other "demo" images too but they also add the sha256 checksum checking. can you add that here too?

Reply:
Good point, I'll add it.